docs/software/ml/index.md

```diff
@@ -0,0 +1,69 @@
+[](){#ref-software-ml}
+# Machine Learning Applications and Frameworks
+
+## Overview
+
+CSCS supports a wide range of machine learning (ML) applications and frameworks
```

Comment:
Not a blocker at all, but FYI: https://eth-cscs.github.io/cscs-docs/contributing/#text-formatting.